SpriterDotNet/Spriter.cs

```diff
@@ -266,17 +211,13 @@ public abstract class SpriterKey : SpriterElement
 
         [XmlAttribute("c4")]
         public float C4;
-
-        public SpriterKey()
-        {
-            Time = 0;
-            CurveType = SpriterCurveType.Linear;
-        }
     }
 
     [Serializable]
     public enum SpriterObjectType
     {
+        None = 0,
```

Comment:
A quick glance at player.scml shows that there are timelines without an object type (line 711 is an example). I don't see any timelines with object_type of "sprite" so I guess Sprite is the default, and I think it should be marked as = 0 to make the default explicit and independent of the order of the values.

As far as making it a nullable enum goes, my hesitance with that is it makes it a bit uglier to work with, and Unity doesn't play nice with nullable primitives (a secondary concern, but still worth thinking about.)